JustSaying.Extensions.DependencyInjection.Microsoft/JustSaying.Extensions.DependencyInjection.Microsoft.csproj

```diff
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\netfx.props" />
```

Comment:
Could simplify by adding this to `Directory.Build.props`, then it's imported into everything automatically?

Reply:
OK, let me peek at that option

Reply:
Doesn't seem to work.

See: https://andrewlock.net/building-net-framework-asp-net-core-apps-on-linux-using-mono-and-the-net-cli/

Reply:
Which bit doesn't work?

Reply:
It doesn't build using the mono framework once you merge the property  group into the DirectoryBuild.Props. See the blog post, the poster there identified that they could not get merging to work. I found the same thing.

Reply:
Ah, right I see.

I thought you were just trying to get it to build the 461 target on not-Windows, rather than trying to get it to build with Mono natively. OK that makes sense then as it wouldn't know to look for the `Directory.Build.props` file to use it at all.

Reply:
Ah I wonder if the issue is MS Build versions for Mono

Reply:
Nah, it is running v15, so it ought to have that feeature. It looks like the global props are not inserted into the build early enough

Reply:
This won't work due to the order msbuild processes the props stuff (it's all a bit black magic). You have to import it as initially done.

Reply:
Also, could we take the approach I added to StatsD:
https://github.com/justeat/JustEat.StatsD/blob/master/ReferenceAssemblies.props

Microsoft are doing some work to make this approach a one-liner.

Reply:
I _knew_ you'd done something somewhere else. I was looking in httpclient-interception and couldn't find it 🤦‍♂️

Reply:
PR: https://github.com/justeat/JustEat.StatsD/pull/164

Reply:
@slang25 @martincostello So you went with the myget approach? I saw that here: https://stu.dev/how-to-target-net4x-on-macos/ and just opted for the other one as I was a little bit cautious of adding a package manager dependency in the file to build on an alternate platform.


Any preferences to how we do this?

Reply:
I'll defer to Stu, but I think I'd prefer whichever one is the smallest diff.

Reply:
LOL. How about I make your change to path, and then push this one as I think that the dependency on myget is a little worse than the 'look up mono if you are on Unix' approach. Then we can change it if we really don't like that, but I can move on to trying to add some value to JustSaying now, whilst we debate the approach :-)

Reply:
I've been preferring the targeting pack from myget, only because it's closest to the solution that will land near the .NET Core 3 time frame.

However both will work, so the smallest diff is a good way to go, we could go with this as it is 👍

Reply:
@slang25 Yeah, it will need review when the .NET Core 3 solution arrives anyway. So...

Reply:
@slang25 Just realized I was using you blog post as the source on how to do it in the first place. LOL.

Reply:
@slang25 
> I've been preferring the targeting pack from myget, only because it's closest to the solution that will land near the .NET Core 3 time frame.

Which solution are you referring to? What's happening near the .NET Core 3 time frame? (I must have missed the memo.)

Reply:
Never mind, [found it](https://github.com/dotnet/designs/pull/33) :)